sympy/solvers/tests/test_ode.py

```diff
@@ -1553,7 +1553,7 @@ def test_1st_homogeneous_coeff_ode2():
     eq1 = f(x).diff(x) - f(x)/x + 1/sin(f(x)/x)
     eq2 = x**2 + f(x)**2 - 2*x*f(x)*f(x).diff(x)
     eq3 = x*exp(f(x)/x) + f(x) - x*f(x).diff(x)
-    sol1 = [Eq(f(x), x*(-acos(C1 + log(x)) + 2*pi)), Eq(f(x), x*acos(C1 + log(x)))]
+    sol1 = Eq(f(x), x*(-acos(C1 + log(x)) + 2*pi))
```

Comment:
Are these equivalent?

Reply:
Yes, I think they are equivalent but I don't know, how I can check it.

Reply:
I'm not sure that they are. I think it's the same solve bug...